### ISSUE TITLE
The 'Getting Started' guide doesn't work. 

### ISSUE THREAD

**Issue by davidgs** [facebook/react-native-website#3915]
## Description

The [documentation](https://reactnative.dev/docs/environment-setup?package-manager=yarn) states: 

> Run the following command to create a new React Native project called "AwesomeProject":
> 
> yarn create expo-app AwesomeProject
> 
> cd AwesomeProject
> yarn expo start
> 
> This will start a development server for you.
> 
> Running your React Native application
> 
> Install the [Expo Go](https://expo.dev/client) app on your iOS or Android phone and connect to the same wireless network as your computer. On Android, use the Expo Go app to scan the QR code from your terminal to open your project. On iOS, use the built-in QR code scanner of the default iOS Camera app.
> 
> Modifying your app
> 
> Now that you have successfully run the app, let's modify it. Open App.js in your text editor of choice and edit some lines. The application should reload automatically once you save your changes.
> 
> That's it!
> 
> Congratulations! You've successfully run and modified your first React Native app.

However, this does not, in fact work. 

The result is:

```
yarn create expo-app AwesomeProject
yarn create v1.22.19
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 🔨  Building fresh packages...
success Installed "create-expo-app@2.1.1" with binaries:
      - create-expo-app
[############################################################################################] 156/156⠋✔ Downloaded and extracted project files.
> yarn install
yarn install v1.22.19
info No lockfile found.
[1/4] 🔍  Resolving packages...
warning expo > uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
warning expo > expo-constants > uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
warning expo > babel-preset-expo > @babel/plugin-proposal-export-namespace-from@7.18.9: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-export-namespace-from instead.
warning expo > babel-preset-expo > @babel/plugin-proposal-object-rest-spread@7.20.7: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-object-rest-spread instead.
warning expo > babel-preset-expo > metro-react-native-babel-preset > @babel/plugin-proposal-object-rest-spread@7.20.7: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-object-rest-spread instead.
warning expo > expo-file-system > uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
warning expo > babel-preset-expo > metro-react-native-babel-preset > @babel/plugin-proposal-async-generator-functions@7.20.7: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-async-generator-functions instead.
warning expo > babel-preset-expo > metro-react-native-babel-preset > @babel/plugin-proposal-class-properties@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-class-properties instead.
warning expo > babel-preset-expo > metro-react-native-babel-preset > @babel/plugin-proposal-numeric-separator@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-numeric-separator instead.
warning expo > babel-preset-expo > metro-react-native-babel-preset > @babel/plugin-proposal-optional-catch-binding@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-optional-catch-binding instead.
warning expo > babel-preset-expo > metro-react-native-babel-preset > @babel/plugin-proposal-nullish-coalescing-operator@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-nullish-coalescing-operator instead.
warning expo > babel-preset-expo > metro-react-native-babel-preset > @babel/plugin-proposal-optional-chaining@7.21.0: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-optional-chaining instead.
warning expo > @expo/cli > cacache > @npmcli/move-file@1.1.2: This functionality has been moved to @npmcli/fs
warning react-native > @react-native/codegen > jscodeshift > @babel/plugin-proposal-class-properties@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-class-properties instead.
warning react-native > @react-native/codegen > jscodeshift > @babel/plugin-proposal-nullish-coalescing-operator@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-nullish-coalescing-operator instead.
warning react-native > @react-native/codegen > jscodeshift > @babel/plugin-proposal-optional-chaining@7.21.0: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-optional-chaining instead.
warning react-native > @react-native-community/cli > @react-native-community/cli-plugin-metro > metro-react-native-babel-transformer > babel-preset-fbjs > @babel/plugin-proposal-class-properties@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-class-properties instead.
warning react-native > @react-native-community/cli > @react-native-community/cli-plugin-metro > metro-react-native-babel-transformer > babel-preset-fbjs > @babel/plugin-proposal-object-rest-spread@7.20.7: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-object-rest-spread instead.
warning react-native > @react-native-community/cli > @react-native-community/cli-plugin-metro > metro > metro-minify-uglify > uglify-es@3.3.9: support for ECMAScript is superseded by `uglify-js` as of v3.13.0
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning "react-native > @react-native/codegen@0.72.7" has unmet peer dependency "@babel/preset-env@^7.1.6".
warning "react-native > @react-native/codegen > jscodeshift@0.14.0" has unmet peer dependency "@babel/preset-env@^7.1.6".
[4/4] 🔨  Building fresh packages...
success Saved lockfile.
✨  Done in 13.73s.

✅ Your project is ready!

To run your project, navigate to the directory and run one of the following yarn commands.

- cd AwesomeProject
- yarn android
- yarn ios
- yarn web
✨  Done in 17.38s.
davidgs@DavidsNewAir github.com % cd AwesomeProject
davidgs@DavidsNewAir AwesomeProject % yarn expo start
yarn run v1.22.19
$ /Users/davidgs/github.com/AwesomeProject/node_modules/.bin/expo start
Starting project at /Users/davidgs/github.com/AwesomeProject
Starting Metro Bundler
▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
█ ▄▄▄▄▄ █ ██▀▀█▄▄██ ▄▄▄▄▄ █
█ █   █ █  ▀█ ▀█ ▄█ █   █ █
█ █▄▄▄█ █▀  █▄▄▀▄██ █▄▄▄█ █
█▄▄▄▄▄▄▄█▄█ ▀▄█▄█ █▄▄▄▄▄▄▄█
█  ▀█  ▄▀▀█▄█▄▀▄▀▄██ ▀▄▄ ▄█
█████▀ ▄▀▄ ▄█▀█▀ ▀▀ █▄  ▀██
█▀ █ ▄ ▄▄▄█▀▄ ▀▄▀▄▀▄▀▀▄ ▀██
███▄▄▄▀▄▄ ▄  ▄▄▄▄▄▄█▄▀ ▀███
█▄▄▄▄█▄▄█▀█▄▀▀█▄▄ ▄▄▄ ▀ ▄▄█
█ ▄▄▄▄▄ █▀▄█ █▄█▀ █▄█ ▀▀▀▀█
█ █   █ █▄   ▀█▄█▄▄ ▄▄▀ ▀▀█
█ █▄▄▄█ █▀ ▀ ███▄██▄▀█▀▀ ██
█▄▄▄▄▄▄▄█▄▄████▄████▄▄▄▄▄▄█

› Metro waiting on exp://192.168.120.216:8081
› Scan the QR code above with Expo Go (Android) or the Camera app (iOS)

› Using Expo Go
› Press s │ switch to development build

› Press a │ open Android
› Press i │ open iOS simulator
› Press w │ open web

› Press j │ open debugger
› Press r │ reload app
› Press m │ toggle menu
› Press o │ open project code in your editor

› Press ? │ show all commands

Logs for your project will appear below. Press Ctrl+C to exit.
Error: Unable to resolve module ./index from /Users/davidgs/github.com/AwesomeProject/.:

None of these files exist:
  * index(.native|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx|.ios.js|.native.js|.js|.ios.jsx|.native.jsx|.jsx|.ios.json|.native.json|.json|.ios.cjs|.native.cjs|.cjs)
  * index/index(.native|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx|.ios.js|.native.js|.js|.ios.jsx|.native.jsx|.jsx|.ios.json|.native.json|.json|.ios.cjs|.native.cjs|.cjs)
    at ModuleResolver.resolveDependency (/Users/davidgs/github.com/AwesomeProject/node_modules/metro/src/node-haste/DependencyGraph/ModuleResolution.js:114:15)
    at DependencyGraph.resolveDependency (/Users/davidgs/github.com/AwesomeProject/node_modules/metro/src/node-haste/DependencyGraph.js:277:43)
    at /Users/davidgs/github.com/AwesomeProject/node_modules/metro/src/lib/transformHelpers.js:169:21
    at Server._resolveRelativePath (/Users/davidgs/github.com/AwesomeProject/node_modules/metro/src/Server.js:1045:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Server._explodedSourceMapForBundleOptions (/Users/davidgs/github.com/AwesomeProject/node_modules/metro/src/Server.js:993:35)
    at async Promise.all (index 0)
    at async Server._symbolicate (/Users/davidgs/github.com/AwesomeProject/node_modules/metro/src/Server.js:945:26)
    at async Server._processRequest (/Users/davidgs/github.com/AwesomeProject/node_modules/metro/src/Server.js:418:7)
```

I guess I would have expected it to install or create all the files. 

## Documentation version

0.72 (latest)

**Comment by Simek** [comment]
Hello @davidgs, unfortunately I cannot reproduce the issue on my machines using macOS and Windows.

Can you run `npx expo-env-info` in the created application directory, and paste the output of that command? This might help us pinpoint the root of issue you are experiencing.

**Comment by cortinico** [comment]
Also this is a Expo issue and should be opened against https://github.com/expo/expo/ and not against the React Native website.
